x-pack/plugins/index_management/public/application/components/mappings_editor/components/document_fields/field_parameters/select_inference_id.tsx

```diff
@@ -111,16 +117,37 @@ export const SelectInferenceId = ({
 
   const fieldConfigModelId = getFieldConfig('inference_id');
   const defaultInferenceIds: EuiSelectableOption[] = useMemo(() => {
-    return [{ checked: 'on', label: 'elser_model_2' }, { label: 'e5' }];
+    return [
+      {
+        checked: 'on',
+        label: i18n.translate(
+          'xpack.idxMgmt.mappingsEditor.parameters.inferenceId.popover.defaultModel.Elser',
+          {
+            defaultMessage: 'elser_model_2',
+          }
+        ),
+        'data-test-subj': 'default-inference_elser_model_2',
+      },
+      {
+        label: i18n.translate(
+          'xpack.idxMgmt.mappingsEditor.parameters.inferenceId.popover.defaultModel.E5',
+          {
+            defaultMessage: 'e5',
```

Comment:
Similar to this [comment](https://github.com/elastic/kibana/pull/184615/files#r1626634568), do we require to translate the model_id 'e5'?